mimic/test/test_maas.py

```diff
@@ -165,11 +165,17 @@ def setUp(self):
         np_id = self.getXobjectIDfromResponse(self.createNotificationPlan('ItsANotificationPlan'))
         sp_id = self.getXobjectIDfromResponse(self.createSuppression('ItsASuppression'))
         self.assertNotEquals(None, entity_id)
+        self.assertTrue(entity_id.startswith('en'))
```

Comment:
Asserts in a `setUp` are generally poor form; when a test fails, an individual test should fail and not a whole suite.  Can you move this setup logic into a thing that can be called from a test, and move the asserts to only *one* test, rather than repeating them in everyone's `setUp`?